sycl/test-e2e/InlineAsm/lit.local.cfg

```diff
@@ -0,0 +1,2 @@
+config.unsupported_features += ['cuda', 'hip']
```

Comment:
IMO, that should be `REQURIES: <intel-gpu>`, although I don't know if that's possible. Also, it might be a good idea to have an extra directory for that (`InlineAsm/IntelGPU`).